code/modules/library/skill_learning/job_skillchips/research_director.dm

```diff
@@ -7,3 +7,6 @@
 	skill_icon = "dumbbell"
 	activate_message = "<span class='notice'>You realise if you apply the correct force, at the correct angle, it is possible to make the immovable permanently movable.</span>"
 	deactivate_message = "<span class='notice'>You forget how to permanently anchor a paradoxical object.</span>"
+	chip_category = SKILLCHIP_CATEGORY_GENERAL
```

Comment:
Why is this a job skillchip if it's not a job chip category?

Reply:
Resolved at commits below